src/main/resources/application.yaml

```diff
@@ -285,6 +300,18 @@ payments:
   api:
     url: http://localhost:8765
 
+serviceRequest:
+  api:
+    callback-url : http://localhost:4000/service-request-update
```

Comment:
I'll admit my ignorance here: I see the same values above. Presuming it's correct, can I ask why?